src/SFA-DAS-HMRC-API-Stub/Commands/GetFractionCalDateRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.HMRC.API.Stub.Commands
+{
+    public class GetFractionCalDateRequest
+    {
+        public DateTime LastCalculationDate;
+
+        public GetFractionCalDateRequest(DateTime lastCalculationDate)

```

Comment:
I don't think the request requires a parameter as none is passed to the Action as far as I can see